webpack.noop.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const webpack = require('webpack'); // eslint-disable-line
+const fileName = 'keen-tracking';
+
+module.exports = {
+  mode: 'development',
```

Comment:
curious what this is used for

Reply:
Good question:  I used the webpack config generator to make most of this and copied some things from the base webpack config.